apps/fluent-tester/src/TestComponents/Badge/BasicBadgeTest.tsx

```diff
@@ -87,6 +91,9 @@ export const BasicBadge: React.FunctionComponent = () => {
           <StyledPicker prompt="Icon position" selected={iconPosition} onChange={onIconPositionChange} collection={badgeIconPositions} />
         </>
       )}
+      <ToggleButton onClick={() => setShowShadow(!showShadow)} checked={showShadow}>
```

Comment:
Seems like this part isn't shown in the screenshots?

Reply:
It is there, it's tint badge on yellow background